.github/workflows/upgrade_version.yaml

```diff
@@ -42,6 +42,9 @@ jobs:
                   # Version in the docs (cargo add slint@<VERSION>):
                   git ls-files | grep "\(^\|/\)docs/.*\.\(md\|rst\)\$" | xargs sed -i 's/slint@[0-9]\+\.[0-9]\+\.[0-9]\+/slint@${{ github.event.inputs.new_version }}/'
 
+                  # Version in the docs (codemirror.js and rollup.config.js):
+                  sed -i 's/[0-9]\+\.[0-9]\+\.[0-9]\+/${{ github.event.inputs.new_version }}/' docs/editor/codemirror.js docs/editor/rollup.config.js
```

Comment:
I don't see the version number in these files.
Is there perhaps another way to get the version?
Should it perhaps be bart of the release script instead like for slintpad?

Reply:
Good catch. This is not needed any longer. Its 'fixed' in build_docs.yaml

sed -i "s/snapshots.slint.dev\/master/releases.slint.dev\/$version/" docs/editor/codemirror.js docs/editor/rollup.config.js